### ISSUE TITLE
Exclude failing to exclude a .swift folder

### ISSUE THREAD

**Issue by negebauer** [realm/jazzy#586]
Version: [`v0.6.3`](https://github.com/realm/jazzy/tree/v0.6.3)

I'm currently using a Swift library called [R.swift](https://github.com/mac-cain13/R.swift) that's located in `myProjectDir/Pods/R.swift`

When running `jazzy` and excluding the `Pods` folder it still tries to read the `R.swift` folder as a `.swift` file, logically failing with this message
```
Could not read contents of `-I/Users/Nico/Documents/Xcode/Wifi UC/Pods/Headers/Public/R.swift`
Could not parse `R.swift`. Please open an issue at https://github.com/jpsim/SourceKitten/issues with the file contents.
building site
jam out ♪♫ to your fresh new docs in `docs`
```

**Comment by jpsim** [comment]
Easy fix, thanks for reporting: jpsim/SourceKitten#220

**Comment by negebauer** [comment]
No problem
Going to check it later

**Comment by negebauer** [comment]
When should this be released?

**Comment by jpsim** [comment]
I just released 0.7.0 now: https://github.com/realm/jazzy/releases/tag/v0.7.0

**Comment by negebauer** [comment]
It's working like a charm! 👍
